pkg/host/store.go

```diff
@@ -221,6 +221,10 @@ func (s *Store[E]) get(id string) (E, error) {
 	}
 
 	obj := s.newFunc()
+	if len(file) == 0 {
```

Comment:
i prefer maybe add mutex.Lock for getting of file content

Reply:
I agree, please use a `RWMutex` in the store.

Reply:
Added.